13/umbraco-deploy/deployment-workflow/import-export.md

```diff
@@ -60,6 +60,7 @@ Similar to when exporting, you can choose to import everything from the archive
 
 {% hint style="info" %}
 Deploy does not touch the default maximum upload size, but you can [configure this yourself by following the CMS documentation](https://docs.umbraco.com/umbraco-cms/reference/configuration/maximumuploadsizesettings).
+However, on Umbraco Cloud the upload size limit is 500mb.
```

Comment:
I would suggest moving this to the CMS documentation, specifically to the existing _"Are you hosting your site on Umbraco Cloud?"_ note...

Reply:
I agree with Ronald, that it makes sense to add it to the note as well :-)

Reply:
I will add the note to the CMS documentation as well :-)